Zyantific.Zydis/Native/Formatter.cs

```diff
@@ -25,11 +123,28 @@ public struct Formatter
             EntryPoint = "ZydisFormatterInit")]
         public static extern ZyanStatus Init(ref Formatter formatter, FormatterStyle style);
 
+        [DllImport(nameof(Zyantific.Zydis), ExactSpelling = true,
+            EntryPoint = "ZydisFormatterSetProperty")]
+        public static extern ZyanStatus SetProperty(ref Formatter formatter, FormatterProperty property, ZyanUPointer value);
+
         [DllImport(nameof(Zyantific.Zydis), ExactSpelling = true, CharSet = CharSet.Ansi,
             EntryPoint = "ZydisFormatterFormatInstruction")]
         public static extern ZyanStatus FormatInstruction(ref Formatter formatter,
             ref DecodedInstruction instruction,
             [MarshalAs(UnmanagedType.LPStr)] StringBuilder buffer, ZyanUSize length,
             ZyanU64 runtimeAddress);
+
+        // The use of UIntPtr as a replacement of size_t may cause issues. There may also be an issue with replacing void* with IntPtrs.
+        [DllImport(nameof(Zyantific.Zydis), ExactSpelling = true,
+            EntryPoint = "ZydisFormatterFormatInstructionEx")]
+        public static extern ZyanStatus FormatInstructionEx(ref Formatter formatter, ref DecodedInstruction instruction, [MarshalAs(UnmanagedType.LPStr)] StringBuilder buffer, System.UIntPtr length, ZyanU64 runtimeAddress, ref System.IntPtr userData);
```

Comment:
- `UIntPtr` should be fine as long as we are compiling for non-segmented architectures (which should always the case)
- Replacing `void*` with `IntPtr` seems to be the common way for C to C# bindings. We might be able to use `void*` in unsafe mode, but not sure about this.